lms/djangoapps/course_api/tests/test_views.py

```diff
@@ -78,6 +77,29 @@ def test_as_honor_for_staff(self):
         self.setup_user(self.honor_user)
         self.verify_response(expected_status_code=403, params={'username': self.staff_user.username})
 
+    def test_filter_by_org(self):
```

Comment:
Is there a way to eliminate the code duplication between test_views and test_api?  Perhaps through a test mixin?